recipes/newrelic/infrastructure/suse.yml

```diff
@@ -9,7 +9,10 @@ repository: https://github.com/newrelic/infrastructure-agent
 installTargets:
   - type: host
     os: linux
-    platform: "suse"
+    platform: suse
```

Comment:
I wonder - is this first installTarget needed? Or would `platformFamily: suse` be enough?

Reply:
I thought about that too, it looks like the older suse populate platform, and wasn't sure if all of those would populate the family as well, since no other linux should be using either with a value of suse, I thought it would be best to have a target for either?